renderlib/command.cpp

```diff
@@ -13,6 +13,20 @@
 #include <errno.h>
 #include <sys/stat.h>
 
+#if defined(WIN32)
+#define STAT64_STRUCT __stat64
+#define STAT64_FUNCTION _stat64
+#elif defined(__APPLE__)
+#define STAT64_STRUCT stat
+#define STAT64_FUNCTION stat
+#elif defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__NetBSD__) || defined(__DragonFly__)
```

Comment:
this is not combined with line 19 to avoid getting Apple cooties on the linux code?  Less comically, so that if other os-dependent stuff needs to happen, you can re-use these blocks.

Reply:
I like keeping the apple and linux sides separate even if they have the same result.  If this kind of thing has to happen more often, it's going to all get moved into some kind of a platform.h file and then reshuffled.  Currently I don't support building on any linux besides Ubuntu, but the stuff I found about 64-bit stat made this distinction clear